### PR TITLE
aiohttp signals integration

### PR DESCRIPTION
**Pull request by jettify**
Code is not working and requires aiohttp from master

### CODE REVIEW

**Comment by adriancole** [comment]
Ah ok. Will be neat to see a micro-intergration test which shows this off once working

**Comment by asvetlov** [comment]
I think task locals usage should be optional, everything should keep working with explicit values.

**Comment by jettify** [comment]
@asvetlov dropped task locals in favor of explicit arguments

File: aiozipkin/aiohttp_helpers.py [hunk 1]
```diff
@@ -89,3 +92,52 @@ def get_tracer(app, tracer_key=APP_AIOZIPKIN_KEY):
...
+
+
+def _has_span(trace_config_ctx):
+    trace_request_ctx = trace_config_ctx.trace_request_ctx
+    has_span = (isinstance(trace_request_ctx, dict) and
+                'span_context' in trace_request_ctx)
+    return has_span
+
+
+def make_trace_config(tracer):
```
Comment (asvetlov) [inline aiozipkin/aiohttp_helpers.py]: It's up to you but the function looks like a class written with JavaScript techniques.

**Comment by jettify** [comment]
I settled with explicit context passing for http client
```python
async def run():
    session = aiohttp.ClientSession(trace_configs=[trace_config])
    ctx = {'span_context': span.context}
    resp = await session.get("http://xkdcd.com", trace_request_ctx=ctx)
    data = await resp.json()
```
Here is example how http client connects with span created by server middleware

https://github.com/aio-libs/aiozipkin/blob/82127f14d53f09030ee6dcdec90fdd46199f591d/demos/microservices/service_a.py#L26-L56

File: aiozipkin/aiohttp_helpers.py (hunk 1 above)
Comment (jettify) [reply aiozipkin/aiohttp_helpers.py]: agreed will fix

Comment (jettify) [reply aiozipkin/aiohttp_helpers.py]: fixed

**Comment by jettify** [comment]
Merging now any other issues will address in next RPs

**Pull request merged**